### DISCUSSION TITLE
Users can develop for an deploy to any cloud

### DISCUSSION THREAD

**Discussion by danmoseley** [microsoft/aspire#2491] [General]

**Comment by dersia** [comment]
it would be great if this would work with the recipes of project radius.

this way you could just select the recipe that would than deploy to "any" cloud.

AFAIK, so far radius supports Azure and AWS (in development)

**Comment by davidfowl** [comment]
I think the model we have been designing with respect to how we’re plugging bicep into aspire might be very compatible with radius. 

Still not a lot of work has gone into figuring that out just yet.
